lib/typing/mongo/bson.go

```diff
@@ -64,12 +65,17 @@ func bsonBinaryValueToMap(value primitive.Binary) (any, error) {
 	case
 		bson.TypeBinaryUUIDOld,
 		bson.TypeBinaryUUID:
-		parsedUUID, err := uuid.FromBytes(value.Data)
-		if err != nil {
-			return nil, err
+		if len(value.Data) == 16 {
```

Comment:
Are there UUIDs in MongoDB that are not 16 bytes or can these fields can be used to store non-UUID values?

Reply:
The latter: can these fields can be used to store non-UUID values